modules/combined/tests/fdp_geometric_coupling/fdp_geometric_coupling_test.i

```diff
@@ -211,4 +211,5 @@
   exodus = true
   print_linear_residuals = true
   print_perf_log = true
+  hide = 'dummy_name_contact_force_0 dummy_name_contact_force_1 dummy_name_contact_force_2'
```

Comment:
Yes, I would appreciate help with that. From what I can tell, hiding is specified and handled by Outputs during initialization, and I did not see a way to change initialization of one Action from within a separate Action.